back/src/main/kotlin/com/sos/smartopenspace/domain/OpenSpace.kt

```diff
@@ -108,11 +109,15 @@ class OpenSpace(
   fun addTalk(talk: Talk): OpenSpace {
     checkIsFinishedQueue()
     checkIsActiveCallForPapers()
+    if (tracksDefined(talk) && !tracks.any { it == talk.track }) throw NotValidTrackForOpenSpaceException()
     talk.openSpace = this
     talks.add(talk)
     return this
   }
 
+  private fun tracksDefined(talk: Talk) =
```

Comment:
le agrega el prefijo is asi queda claro que es un boolean y no una propiedad con un valor.